bindings/py/cpp_src/bindings/algorithms/py_SpatialPooler.cpp

```diff
@@ -225,6 +225,14 @@ Argument wrapAround boolean value that determines whether or not inputs
         py_SpatialPooler.def("getMinPctOverlapDutyCycles", &SpatialPooler::getMinPctOverlapDutyCycles);
         py_SpatialPooler.def("setMinPctOverlapDutyCycles", &SpatialPooler::setMinPctOverlapDutyCycles);
 
+	py_SpatialPooler.def("anomaly", [](const SpatialPooler& self) {
+			return self.anomaly;
+	});
+	py_SpatialPooler.def("anomalyThreshold", [](const SpatialPooler& self) {
+			return &self.spAnomaly.SPATIAL_TOLERANCE;
```

Comment:
added the py bindings. Will this work so that I'll be able to set through the variable, or is it just a getter?